### PR TITLE
Release 1.5 postperations

### PR DESCRIPTION
**Pull request by roypat**
Things that need to happen after the release branch was taken

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following
Developer Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

- [ ] If a specific issue led to this PR, this PR closes the issue.
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] API changes follow the [Runbook for Firecracker API changes][2].
- [ ] User-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.
- [ ] New `TODO`s link to an issue.
- [ ] Commits meet [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

---

- [ ] This functionality cannot be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

**Review by pb8o — APPROVED** [review]

File: tools/devtool
Comment (roypat) [reply tools/devtool:523]: It passed yesterday, lets hope it still does :pray:

Comment (roypat) [reply tools/devtool:523]: and by yesterday I mean Friday

Comment (pb8o) [reply tools/devtool:523]: OK c7g / host=4.14 still needs the guest 5.10 legacy artifact. We can try the no-sve version if it works, otherwise we can copy over the v1.5 version of it.

Comment (pb8o) [reply tools/devtool:523]: All tests are skipped in https://buildkite.com/firecracker/firecracker-pr/builds/6599#018b14f8-41fb-40aa-a795-210362c942a5/55-116

Comment (roypat) [reply tools/devtool:523]: Fixed up the artifacts, seems like the c7g tests are no longer skipped now (copied over the `legacy` folder from the 1.5 artifacts)

**Review by zulinx86 — DISMISSED** [review]

**Comment by roypat** [comment]
> nit: regarding terminology in the commit message, we're still using a single S3 bucket with per-release folders.

fixed :) thankfully we can update commit messages without dismissing approvals lol

**Review by pb8o — APPROVED** [review]

**Pull request merged**
